components/enterprise-contract/base/tekton-chains/kustomization.yaml

```diff
@@ -25,5 +27,6 @@ patchesStrategicMerge:
 # Mount the shared secret
 - tekton-chains-controller-shared-secret.yaml
 
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
+patches:
+# Remove the chains-config
+- remove-chains-config.yaml
```

Comment:
Why do we do this?

Reply:
Because we don't ArgoCD to load the chains-config ConfigMap from the Helm charts and from the configuration in the `rekor-*/tekton-chains-config.yaml`. Having two sources for the same object makes ArgoCD flip-flop between them